products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -456,6 +476,11 @@ public final class BuildSystem {
         // Construct the system delegate.
         var _delegate = llb_buildsystem_delegate_t()
         _delegate.context = Unmanaged.passUnretained(self).toOpaque()
+        if delegate.fs != nil {
+            _delegate.fs_get_file_contents = { BuildSystem.toSystem($0!).fsGetFileContents(String(cString: $1!), $2!) }
+            _delegate.fs_get_file_info = { BuildSystem.toSystem($0!).fsGetFileInfo(String(cString: $1!), $2!) }
+            _delegate.fs_get_link_info = { BuildSystem.toSystem($0!).fsGetFileInfo(String(cString: $1!), $2!) }
```

Comment:
This should be a separate callback, not shared with getFileInfo (or get FileInfo should take a parameter)

Reply:
Added as FIXME for now.